packages/node_modules/@webex/plugin-meetings/src/meeting/request.js

```diff
@@ -43,6 +43,11 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       sipUri, deviceUrl, locusUrl, resourceId, correlationId, ensureConversation, moderator, pin, moveToResource, roapMessage
     } = options;
 
+    LoggerProxy.logger.log(
```

Comment:
Would prefer these request logs to be `.info` because I don't think the normal sdk user will need them, but it isn't blocking. Could be addressed at a later date.